L1Trigger/VertexFinder/src/VertexFinder.cc

```diff
@@ -986,29 +997,46 @@ namespace l1tVertexFinder {
                                          << "\n"
                                          << "tkZ0 = " << tkZ0.to_double() << "(" << tkZ0.to_string(2)
                                          << ")\ttkpt = " << tkpt.to_double() << "(" << tkpt.to_string(2)
-                                         << ")\tpt_tmp = " << pt_tmp << "\tbin = " << bin.first.to_int() << "\n"
+                                         << ")\tbin = " << bin.first.to_int() << "\n"
```

Comment:
I would say that these `LogInfo` instances should be under `LogDebug`, but given that they are controlled by a debug parameter and not all of it is introduced here, it is not urgent.

Reply:
This LogInfo has been used pretty frequently by the previous developers. I have no opposition to changing it (off the top of my head), but like the constants, should probably be coordinated across all the GTT modules at once.